vignettes/getting-started.Rmd

```diff
@@ -299,6 +321,16 @@ mod4 <- copy_model_from(mod1, "4") %>%
   add_notes("Initial estimates inherited from mod1 results")
 ```
 
+You can also tweak or 'jitter' the initial parameter estimates using `tweak_initial_estimates()`. This can be used both in conjunction with `inherit_param_estimates()` or as a standalone call:
+```{r tweak_initial_estimates}
+mod5 <- copy_model_from(mod1, "5") %>% 
+  inherit_param_estimates(digits = 3) %>% 
+  tweak_initial_estimates(digits = 3, .p = 0.2)
+
+mod6 <- copy_model_from(mod1, "6") %>% 
+  tweak_initial_estimates(digits = 3, .p = 0.2)
```

Comment:
@seth127 let me know if you'd prefer to ditch this example. I thought it would be nice to mention without adding too many details.